app/helpers/application_helper.rb

```diff
@@ -39,6 +39,14 @@ def available_sign_up_path
     end
   end
 
+  def max_bio_chars
+    Setting.max_bio_chars
+  end
+
+  def max_bio_chars
+    Setting.max_bio_chars
+  end
```

Comment:
Is there supposed to be 2 copies of the same function?

Reply:
No, looks like a bad merge.  I'll remove the dupe